src/bundle/Resources/public/js/scripts/core/base.chart.js

```diff
@@ -42,9 +42,6 @@
         setOptions(options) {
             this.options = {
                 ...defaultOptions,
-                animation: {
-                    onComplete: this.onCompleteAnimationCallback,
```

Comment:
Note: it did not work as expected as it was also triggered when animating these tooltips:
![Screenshot 2021-11-02 at 11 54 42](https://user-images.githubusercontent.com/10233057/139833979-1a51a61a-b4f6-45b7-b259-d7e3380a2ce0.png)
It was deleted as there is no need for it.